.travis.yml

```diff
@@ -70,17 +73,27 @@ cache:
   - ${HOME}/opt
 
 before_install:
-  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then brew update     ; fi
-  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then brew install lzo; fi
+  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
+      brew update;
+      brew install lzo;
+    fi
 
 install:
   - .travis/build-deps.sh > build-deps.log 2>&1 || (cat build-deps.log && exit 1)
+  - if [ ! -z "${CHOST+xxx}" ]; then unset CC; unset CXX; fi
```

Comment:
I haven't seen this `+xxx` before.  What does it do?  Is it some bash trick I should know?